src/constants.ts

```diff
@@ -10,6 +10,8 @@ export const EDIT_BLUEPRINT = `${IMAGE_BUILDER_API}/blueprints`;
 
 export const CDN_PROD_URL = 'https://cdn.redhat.com/';
 export const CDN_STAGE_URL = 'https://cdn.stage.redhat.com/';
+export const CONTENT_BETA = '/preview/insights/content';
+export const CONTENT_STABLE = '/insights/content';
```

Comment:
I'm not pushing for this, but would one constant with a function betaUrl(path) => `/review/${path}` be something we would like to entertain?

Reply:
Great idea! I've added a new commit, let me know if it looks like what you've had in mind.

Since BetaPath uses a `useGetEnvironment()` hook, it needs to be either a custom hook or a React component I think. And it's called conditionally in some places so I've made it a component as a custom hook made the linter unhappy.